Swerve/SwerveModules/SwerveModuleNeo.java

```diff
@@ -45,7 +49,26 @@ protected Distance getPosition() {
 
     @Override
     protected LinearVelocity getVelocity() {
-        return Units.MetersPerSecond.of(driveMotor.getEncoder().getVelocity());
+        // Convert from default units RPM to M/S
+        return Units.MetersPerSecond.of(driveMotor.getEncoder().getVelocity() *
```

Comment:
I would double check the units we set to the neo motor, don't think we have it in RPM?

Reply:
The unit we convert to is stored in the conversionConfig parameter of the motorConfig constructor, which in the constants folder in frc2025 is left to its default value of Rotations. In conversionConfig, the value of RPM is divided by 60 to change to seconds, so I removed the division by 60 for the neo velocity